docs/tutorials/fabric_remote.md

```diff
@@ -0,0 +1,425 @@
+---
+layout: default
+title: Work with remote fabric network
+parent: pages.tutorials
+nav_order:
+---
+
+# Work with remote Hyperledger Fabric Network
+
+{: .no_toc }
+
+This guide describes the steps to use FireFly to interact with a chaincode deployed to an external Hyperledger Fabric blockchain network (not created by Firefly) in order to submit transactions and query for states.
```

Comment:
I think adding a note at the top of this tutorial to set a bit more context would be helpful to people. While this is a fantastic guide for helping people connect a local development environment to a pre-existing Fabric network, we don't want people to get the idea that the FireFly CLI should be used for setting up production networks. I might suggest something like this:

```suggestion
The FireFly CLI makes it quick and easy to create an entire FireFly development environment on your local machine, including a new blockchain from scratch. However, sometimes a developer may want to connect their local FireFly development environment to a Fabric network that already exists on their machine or elsewhere. This guide describes the steps to connect FireFly to an external Hyperledger Fabric blockchain network (not created by the FireFly CLI), including interacting with a custom chaincode in order to submit transactions and query for state.
```